### ISSUE TITLE
CupboardsCloseAll  Bug Fix

### ISSUE THREAD

**Issue by fangqi-Zhu** [chernyadev/bigym#17]
![1724488205916](https://github.com/user-attachments/assets/b1de2706-3332-4899-be0a-319de4841c29)

CupboardsCloseAll: It seems there are some issues with this environment, can I request a fix for this environment?

**Comment by chernyadev** [comment]
Hi @fangqi-Zhu,

Thanks for reporting this. Unfortunately can't reproduce the issue shown in the screenshot. Could you provide more details, such as the platform you're using, the code/configuration, and the steps to reproduce the problem? This will help us investigate further.

Thank you!

![image](https://github.com/user-attachments/assets/87295e9f-7bb1-4b27-83fa-6310167ba7c7)

**Comment by fangqi-Zhu** [comment]
Thank you for your confirmation!  
I was able to reproduce this issue on both macOS and Windows operating systems without modifying the code. You can reproduce the issue using the following code:

```
"""An example of using BiGym with state."""
from bigym.action_modes import JointPositionActionMode
from bigym.envs.reach_target import ReachTarget
from bigym.envs.cupboards import (
    DrawerTopOpen,
    DrawerTopClose,
    DrawersAllOpen,
    DrawersAllClose,
    WallCupboardOpen,
    WallCupboardClose,
    CupboardsOpenAll,
    CupboardsCloseAll,
)
 
from bigym.robots.configs.h1 import H1 

print("Running 1000 steps without pixels...")
env = DrawersAllOpen(
    action_mode=JointPositionActionMode(floating_base=True, absolute=True, block_until_reached=False),
    render_mode="human",
    robot_cls = H1
)

print("Observation Space:")
print(env.observation_space)
print("Action Space:")
print(env.action_space)

env.reset()
for i in range(100000):
    actions = env.action_space.sample()
    obs, reward, terminated, truncated, info = env.step(actions)
    img = env.render()
env.close()
```


I found that when I modified the Euler parameters in bigym/envs/presets/counter_base_wall_3x1.yaml, I was able to rotate the BaseCabinet, but the WallCabinet could not be rotated. I'm not sure if you can reproduce this issue. Thank you very much for your help.

**Comment by chernyadev** [comment]
Thanks for sharing the code and details!

I haven’t been able to reproduce the issue on Linux, but I’ll check it out on macOS and update you here.

**Comment by chernyadev** [comment]
Quick update on the issue, it is related to updates of the Mojo, we are working on fixing these problem, it will be fixed in https://github.com/chernyadev/bigym/pull/19.

**Comment by chernyadev** [comment]
The issue should be resolved in #19. Could you please confirm on your side?

**Comment by fangqi-Zhu** [comment]
Thank you for your kind help!
